src/main/java/com/iexec/core/replicate/ReplicatesList.java

```diff
@@ -43,12 +43,12 @@
 
     private List<Replicate> replicates;
 
-    ReplicatesList(String chainTaskId) {
+    public ReplicatesList(String chainTaskId) {
         this.chainTaskId = chainTaskId;
         this.replicates = new ArrayList<>();
     }
 
-    ReplicatesList(String chainTaskId, List<Replicate> replicates) {
+    public ReplicatesList(String chainTaskId, List<Replicate> replicates) {
```

Comment:
That one is useless

Reply:
Sure, but I find it better to be consistent with the other constructor.